content/en/tracing/setup_overview/custom_instrumentation/dotnet.md

```diff
@@ -119,9 +119,59 @@ using (var parentScope =
     }
 }
 ```
+
+### Headers extraction and injection
+
+Datadog APM tracer supports [B3][5] and [W3C][6] headers extraction and injection for distributed tracing. You can find more details on this in the [setup documentation][7].
+
+Headers extraction and injection will be transparent in most cases.
+Though, in some rare cases the span context can be lost and only a custom implementation will be able to fix it.
+
+To do so, please do the following:
```

Comment:
```suggestion
In most cases, the header's extraction and injection are transparent. If your span context gets lost, create a custom implementation.
```

Reply:
🤔 , I think I need to explicit more when this can happen.
I think I'll rework it to:

> In most cases, the header's extraction and injection are transparent. Though, there are some known cases where your distribute trace can be disconnected. For instance, when reading messages from a distributed queue, some library will lose the span context. In that case, you can add a custom trace using the following code:

WDYT?